### DISCUSSION TITLE
我想请教一下怎么快速修改配置参数

### DISCUSSION THREAD

**Discussion by andlink01** [kkkgo/PaoPaoDNS#231] [问答和帮助] [answered]
我创建容器的时候用的第一个docker模板命令只有CNAUTO=yes，其他的都是默认值。怎么修改其他参数

**Comment by xingshu1990** [comment, marked as answer]
大佬们有自己的精神洁癖，死犟着不发布自己的配置，
想当然的认为，多数人只看wiki，就能拥有举一反三的能力。
https://github.com/kkkgo/PaoPaoDNS/discussions/categories/%E7%BB%8F%E9%AA%8C%E5%88%86%E4%BA%AB
这个版块下，有多人发布的内容中，包含了配置参数。
建议让chatgpt等AI程序对wiki等内容 进行分析，然后自行决定是否写入到配置里。

**Reply by andlink01** [reply]
谢谢你，我是想问已经创建的容器，怎么修改配置？总不能为了一个参数删除容器重建吧

**Reply by xingshu1990** [reply]
> 谢谢你，我是想问已经创建的容器，怎么修改配置？总不能为了一个参数删除容器重建吧

docker下不知道，看看docker有什么启动容器时，能加载配置文件的相关配置。我的笨办法就是，改用docker-compose

**Comment by kkkgo** [comment]
文档中关于“custom_env.ini”的描述符合你的需求。
custom_env.ini可以自定义环境变量，会覆盖在容器在启动时的环境变量。在容器启动后修改该文件将会导致MosDNS重载，但在容器启动后修改的环境变量不会影响已经启动的其他组件。配置的格式为key="value"（注意英文双引号），错误格式的环境变量将会被忽略加载。容器版本更新不会覆盖该文件。
